pre_commit/commands/run.py

```diff
@@ -48,8 +49,21 @@ def _print_user_skipped(hook, write, args):
     ))
 
 
+def get_changed_files(new, old):
+    changed_files = cmd_output(
+        'git', 'show', '--pretty=format:', '--name-only',
+        '{0}..{1}'.format(old, new))[1].splitlines()
```

Comment:
we prefer this style when making multiline invocations:

```
func(
    params...,
)
```

So this one would look like:

```
changed_files = cmd_output(
    'git', 'diff', '--name-only', '{0}..{1}'.format(old, new),
)[0].splitlines()
```